docs/content/latest/integrations/django-rest-framework.md

```diff
@@ -0,0 +1,197 @@
+        model = Users
+        fields = ('userId', 'firstName', 'lastName', 'email')
+```
+
+- Create a [ViewSet](https://www.django-rest-framework.org/api-guide/viewsets/) in `testdb/views.py` and add the following code:
+
+```python
+from django.shortcuts import render
+from testdb.models import Users
+from rest_framework import viewsets
+from testdb.serializers import UserSerializer
+
+class UserViewSet(viewsets.ModelViewSet):
+    queryset = Users.objects.all()
+    serializer_class = UserSerializer
+```
+
+The application now has all the elements and is ready to be configured as follows:
+
+- Map the URLs in `yugabyteTest/urls.py` by adding the following code:
+
+```python
+from django.urls import include, re_path
+from rest_framework import routers
+from testdb.views import UserViewSet
+
+router = routers.SimpleRouter(trailing_slash=False)
+router.register(r'users', UserViewSet)
+urlpatterns = [
+    re_path(r'^', include(router.urls))
+]
+```
+
+For Django versions below 4.0 add the following code in `urls.py` instead, since you can import the urls using `django.conf.urls`:
+
+```python
+from django.urls import path, include
+from django.conf.urls import url, include
+from rest_framework import routers
+from testdb.views import UserViewSet
+
+router = routers.SimpleRouter(trailing_slash=False)
+router.register(r'users', UserViewSet)
+
+urlpatterns = [
+    url(r'^', include(router.urls))
+]
+```
+
+- Configure the `yugabyteTest/settings.py` to use YugabyteDB by updating the value of DATABASES field with:
+
+```python
+DATABASES = {
+    'default': {
+        'ENGINE': 'yb_backend',
+        'NAME': 'yugabyte',
+        'HOST': 'localhost',
+        'PORT': 5433,
+        'USER': 'yugabyte',
+        'PASSWORD': 'yugabyte'
+    }
+}
+```
+
+You also need the application and rest framework in the `INSTALLED_APPS` field. Replace the existing code with:
+
+```python
+INSTALLED_APPS = [
+    'rest_framework',
+    'testdb.apps.TestdbConfig',
+    'django.contrib.contenttypes',
+    'django.contrib.auth',
+]
+
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': [],
+    'DEFAULT_PERMISSION_CLASSES': [],
+    'UNAUTHENTICATED_USER': None,
+}
+```
+
+- Create the migrations with the command:
+
+```python
+python3 manage.py makemigrations
+```
+
+- Migrate the changes to the database with the command:
+
+```python
+python3 manage.py migrate
+```
+
+- A users table should be created in your database. Verify that the table is created with the [ysqlsh](../../explore/ysql-language-features/databases-schemas-tables/#list-tables) client shell.
+
+## Run the application
+
+- Run the django project with the command:
+
+```python
+python3 manage.py runserver 8080
+```
+
+- Insert a row with the command:
+
+```sh
+$ curl --data '{ "firstName" : "John", "lastName" : "Smith", "email" : "jsmith@yb.com" }' \
+       -v -X POST -H 'Content-Type:application/json' http://localhost:8080/users
+```
+
+- Verify that the new row is inserted with the command:
```

Comment:
Suggestion: 
Verify that the new row has been inserted by executing the following command: